.github/workflows/bump_project_version.yml

```diff
@@ -0,0 +1,22 @@
+#############################################################################
+# GitHub Action to Bump Project Version.
+#
+# Workflow starts every day at the end of the day.
+#
+#############################################################################
+name: "Bump Project Version"
```

Comment:
fix name and apply all comment from https://github.com/checkstyle/contribution/pull/635#pullrequestreview-1079599409

Reply:
Changes Done